### DISCUSSION TITLE
Rename Enum Variants

### DISCUSSION THREAD

**Discussion by yodaldevoid** [PyO3/pyo3#2455] [Questions] [answered]
Is there currently any way to expose a different name for an enum variant to python from the one used in Rust? I like to use names that fit the standard Python formatting from my Python API, but this means that I either need to have enum variants that don't match the Rust formatting throughout my Rust code, or I need to duplicate each enum to have Rust version and a Python version.

**Comment by davidhewitt** [comment, marked as answer]
I would accept a PR to add `#[pyo3(name = "...")]` support for enum variants!

**Reply by yodaldevoid** [reply]
Fair enough #2457
